atom/browser/resources/win/atom.manifest

```diff
@@ -32,7 +32,6 @@
 
   <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
     <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
-      <dpiAware>true</dpiAware>
```

Comment:
You may also want to look at what we added in Windows 10 Creators update which is detailed here: https://blogs.windows.com/buildingapps/2017/04/04/high-dpi-scaling-improvements-desktop-applications-windows-10-creators-update/#HBK3Qfrm4qbzWAEV.97